### DISCUSSION TITLE
Need help with installing OCI8

### DISCUSSION THREAD

**Discussion by PrasadChinwal** [php/frankenphp#703] [Q&A] [answered]
I am facing some issues with installing oci8 for oracle. Specifically the extension `pdo_oci`
Anybody has any steps or a sample dockerfile that I could refer?

The apps that we build heavily rely on oracle and that's the only piece we are missing. Any help is appreciated. Thanks!

**Comment by speich** [comment]
I would be interested to know as well.

**Reply by AlliBalliBaba** [reply, marked as answer]
Did this not work for you?

```dockerfile
FROM dunglas/frankenphp

# add additional extensions here:
RUN install-php-extensions \
	pdo_oci
```
